applications/GeoMechanicsApplication/custom_elements/axisymmetric_stress_state.cpp

```diff
@@ -17,11 +17,11 @@
 namespace Kratos
 {
 
-Matrix AxisymmetricStressState::CalculateBMatrix(const Matrix&         rGradNpT,
-                                                 const Vector&         rNp,
+Matrix AxisymmetricStressState::CalculateBMatrix(const Matrix&         rDN_DX,
```

Comment:
This is better, I did not understand the p in the old name GradNpT.
An alternative would have been GradNT ( I guess the T is for transposed ) DX also refers to a spatial gradient.